components/map/map.jsx

```diff
@@ -10,61 +16,146 @@ const POKE_MARKERS = [
   'https://res.cloudinary.com/joycechau/image/upload/v1487572666/premierball.png'
 ];
 
-const MAX_LAT = 49.5;
-const MIN_LAT = 17.5;
-const MAX_LNG = 294.5;
-const MIN_LNG = 236;
+const MAX_LAT = 85;
+const MIN_LAT = -85;
+const MAX_LNG = 180;
+const MIN_LNG = -180;
 
 export default class Map extends React.Component {
   constructor(props) {
     super(props);
+    this.state = { map: null, pokemon: null, modalOpen: false };
```

Comment:
I don't think map is being used in the rendering state. You only need to store variables in the state if the component depends on it to be rendered. So it's fine to store your map in an instance variable i.e this._map or something if you need to save it somewhere